### PR TITLE
[IMP] convert_field_to_html: be ablet to avoid spamming query for each row

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
In v15, this method is used several times and some times in tables that usually have many rows.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Uhm, the operation is not the same right now...

You can add a `verbose=True` keyword argument and don't log if verbose=False.

**Comment by MiquelRForgeFlow** [comment]
> Uhm, the operation is not the same right now...

What do you mean?

**Comment by pedrobaeza** [comment]
You are not processing each content and pass `plaintext2html` over it.

**Comment by MiquelRForgeFlow** [comment]
ah, shit. right!

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -2195,12 +2195,18 @@ def convert_field_to_html(cr, table, field_name, html_field_name):
...
-            }, (plaintext2html(row[1]), row[0])
-        )
+        query = "UPDATE %(table)s SET %(field)s = %%s WHERE id = %%s" % {
+            'field': html_field_name,
+            'table': table,
+        }
+        if verbose:
+            logged_query(
+                cr, query, (plaintext2html(row[1]), row[0])
+            )
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py:2205]: Everything fits in one line:

```suggestion
            logged_query(cr, query, (plaintext2html(row[1]), row[0]))
```

File: openupgradelib/openupgrade.py [hunk 2]
```diff
@@ -2195,12 +2195,18 @@ def convert_field_to_html(cr, table, field_name, html_field_name):
...
+        else:
+            cr.execute(
+                query, (plaintext2html(row[1]), row[0])
+            )
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py:2209]:
```suggestion
            cr.execute(query, (plaintext2html(row[1]), row[0]))
```

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**